tools/cxbe/Exe.cpp

```diff
@@ -158,6 +158,8 @@ Exe::Exe(const char *x_szFilename)
         }
     }
 
+    m_debugPath = x_szFilename;
```

Comment:
I think I'd prefer if this was an optional thing or user-controlled.

I'm concerned with leaking user-data (such as usernames) or storing unexpected data (which XBE parsers can't handle - such as non Windows paths).

Reply:
Added `DEBUGPATH` option.